### PR TITLE
Pom file updation.

### PR DESCRIPTION
**Pull request by akshaysharma096**
1. Add fix to update POM file.
2. Added fix for importing modules in __init__.py
3. Moved code to PEP8 format.
4. Updated `setup.py`

### CODE REVIEW

**Comment by akshaysharma096** [comment]
@maxpumperla @agibsonccc

File: jumpy/ndarray.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def _from_numpy(np_array):
     # Convert the numpy array to nd4j context dtype
     required_dtype = get_np_dtype(get_context_dtype())
     if np_array.dtype != required_dtype:
-        raise Exception(required_dtype + ' required.')
+        raise Exception("%s is requied." % repr(np_array.dtype))
```
Comment (maxpumperla) [inline jumpy/ndarray.py]: typo: "requied" ==> "required"

also, I favour `"foo {} bar".format(x)` notation, but that's just taste, I guess

**Comment by akshaysharma096** [comment]
@farizrahman4u @maxpumperla please have a look, thank you!

**Comment by farizrahman4u** [comment]
LGTM

File: pom.xml [hunk 2]
```diff
@@ -4,25 +4,15 @@
     <groupId>org.deeplearning4j</groupId>
     <artifactId>dl4j</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0-alpha</version>
```
Comment (saudet) [inline pom.xml]: BTW, the latest version is still 1.0.0-SNAPSHOT, not 1.0.0-alpha...

Comment (akshaysharma096) [reply pom.xml]: @saudet , @agibsonccc specially mentioned to keep it alpha.

**Comment by akshaysharma096** [comment]
@agibsonccc  @maxpumperla can we merge the PR?

**Pull request merged**